Allow to use any template by giving its path (#78)
HaxeFlixel/flixel-tools: info/template.txt

>  Usage : template <name> <path> <options>

Author: ```suggestion
 Usage : template <tpl-name | tpl-path> <target-path> <options>
```

>   <name>    : The name or the path for the template to generate. 
>               The "default" template will be created when none is specified.

Author: ```suggestion
  <tpl-name | tpl-path> :
              The name or the path of the template to generate.
              The "default" template will be created when none is specified.
```

>   <path>    : The name for the path to create the template in, default
>               is the name of the template itself.

Author: ```suggestion
  <target-path> :
              The path to create the template in, default is the name
              of the template itself.
```
